src/templates/app/_layout/layout.jsx

```diff
@@ -72,6 +73,14 @@ const Layout = () => {
                 <InterviewPopup /> {/* TODO: remove when UX research is finished */}
                 <DerivIFrame />
                 {/* <Elevio /> */}
+                <div id='language-menu-modal' className='invisible'>
+                    <div id='language-menu'>
+                        <div id='language-menu-header'>
+                            <h3>{it.L('Language Settings')}</h3>
```

Comment:
This should be sentence case as per Deriv.app's styling guide. Please inform the design team if the design shows it in _Title Case_.

```suggestion
                            <h3>{it.L('Language settings')}</h3>
```